extensions/measurement-tracking/src/panels/PanelStudyBrowserTracking/PanelStudyBrowserTracking.tsx

```diff
@@ -24,6 +30,35 @@ const thumbnailNoImageModalities = [
   'OT',
   'PMAP',
 ];
+
+const getMenuItems = ({ commandsManager, items, servicesManager, ...props }) => {
```

Comment:
Can the menu items come from the base study browser?  Is there a good reason to use different menus/setups between the tracked and non-tracked study browser?